fetch.bs

```diff
@@ -5787,6 +5798,11 @@ initially a new {{AbortSignal}} object.
  <dd>Returns a boolean indicating whether or not <var>request</var> can outlive the global in which
  it was created.
 
+ <dt><code><var>request</var> . <a attribute for=Request>speculative</a></code>
+ <dd>Returns a boolean indicating whether or not <var>request</var> is a speculative fetch, which
+ means it could be ignored by the user agent, have low priority, and can outlive the global in which
```

Comment:
What does ignored mean? Would the promise never resolve?

Reply:
I guess it would be better if the promise got rejected in those cases. That would probably require adding a step to https://fetch.spec.whatwg.org/#concept-fetch which actually does that.